src/codeu/chat/codeU_db/DropDatabase.java

```diff
@@ -0,0 +1,11 @@
+package codeu.chat.codeU_db;
+
+public class DropDatabase
+{
+    public static void main( String args[] )
+    {
+        DataBaseConnection.open();
+        DataBaseConnection.dropTables();
+        System.out.println("Finished Droping Database Successfully!!!");
```

Comment:
There should be a DataBaseConnection.close() here and in the other main functions.